.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -7,22 +7,17 @@ body:
       value: |
         Thank you for contributing to the Klaviyo Android SDK!
         
-        Before you submit your issue, please complete each text area below with the relevant details
-        for your bug, and complete the steps in the checklist.
+        Before you submit your issue, please complete each text area below with the relevant details for your bug, and complete the steps in the checklist.
         
-        We welcome your input! If you have any code suggestions regarding your issue, feel free to 
-        [submit a pull request](https://github.com/klaviyo/klaviyo-android-sdk/pulls) 
-        in addition to creating an issue.
+        We welcome your input! If you have any code suggestions regarding your issue, feel free to [submit a pull request](https://github.com/klaviyo/klaviyo-android-sdk/pulls) after creating an issue.
```

Comment:
i had put linebreaks in here for readability in the IDE, usually markdown ignores that, but not within yml so whatever.